apps/dashboard/app/(app)/apis/actions.ts

```diff
@@ -30,8 +31,23 @@ export async function fetchApiOverview({
     with: {
       keyAuth: {
         columns: {
+          id: true, // Include the keyspace ID
           sizeApprox: true,
         },
+        with: {
+          keys: {
```

Comment:
I'm not sure I understand why we need to load keys here

Reply:
I was experimenting with loading keys probably forgot to remove it. It should not be there you are right